src/fixtures/beacons.fixture.ts

```diff
@@ -8,14 +8,14 @@ export const beaconFixture: Readonly<IBeacon> = Object.freeze({
   hexId: "1D0EA08C52FFBFF",
   type: BeaconTypes.Epirb,
   protocolCode: "AX7098",
-  registeredDate: "2018-06-08T00:00",
+  registeredDate: "2018-06-08",
```

Comment:
`BeaconApiResponseMapper` now strips timezone information from the date fields at the boundary.